src/Microsoft.Identity.Client/DefaultEvent.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Identity.Client.Internal;
```

Comment:
Hmm, most of the time VisualStudio feels them in automatically, but not always. :-/  Will do.